code/menuui/credits.h

```diff
@@ -12,10 +12,28 @@
 #ifndef __CREDITS_H__
 #define __CREDITS_H__
 
+struct credits_info {
+	SCP_string music;						// the mission name
+	int num_images;							// the mission filename
+	int start_index;						// the mission description
+	float scroll_rate;						// the mission designer
+	float art_display_time;					// the mission designer
+	float art_fade_time;					// the mission designer
+	SCP_vector<SCP_string> credit_parts;	// the lines of credits
+	SCP_string credits_complete;			// the lines of credits
+};
+
+extern credits_info Credits_Info;
```

Comment:
I have to ask: What do you need this struct for? When are these values ever different from the globals that you use to fill these up with data?

Reply:
Fixed per our discussion